Bottom: e2a20bb253bb4adce12436705eebac81a0a8089d
Top:    88943c6c894190c73169df45798c421280bad3c1
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-06-29 16:44:41 +0800

Refresh of tun-introduce-tx-array-mode

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index ed76128..0a24857 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -1659,8 +1659,7 @@ static struct proto tun_proto = {
 
 static int tun_flags(struct tun_struct *tun)
 {
-	return tun->flags & (TUN_FEATURES | IFF_PERSIST | IFF_TUN |
-			     IFF_TAP);
+	return tun->flags & (TUN_FEATURES | IFF_PERSIST | IFF_TUN | IFF_TAP);
 }
 
 static ssize_t tun_show_flags(struct device *dev, struct device_attribute *attr,
